edgemesh/pkg/dns/dns.go

```diff
@@ -77,29 +77,47 @@ var metaClient client.CoreInterface
 // dnsConn saves DNS protocol
 var dnsConn *net.UDPConn
 
+//Init creates network interface
+func Init(addr string) {
+	// create a network interface
+	checkInterface := fmt.Sprintf("brctl show | grep %s 2>/dev/null", ifi)
```

Comment:
Is it possible to use [netlink ](https://github.com/vishvananda/netlink) instead of system command?